.travis.yml

```diff
@@ -59,7 +59,7 @@ env:
     - PGVERSION=9.0 PQTEST_BINARY_PARAMETERS=no
 
 script:
- - go test -v ./...
+ - result=$(gofmt -d -e $(find -name \*.go)); test -z "$result" || (echo "$result" && false) && go test -v ./...
```

Comment:
Ah, okay. I'm not going to tweak this further since it works and I don't feel like playing Travis golf ;) but if you want to refine it, I'll accept a PR.